tests/contrib/yaaredis/test_yaaredis.py

```diff
@@ -0,0 +1,125 @@
+# -*- encoding: utf-8 -*-
```

Comment:
yeah - we gotta make sure that the integration implements the logic for the integration config and for the pin so that the service name can be overridden if desired.

Reply:
Ok — just to be clear, those tests I think come from both `aredis` and `redis` who don't have it then.

Reply:
yeah they _should_ though 😕